apps/api/jwt_auth/views.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import exceptions
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.views import APIView
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_jwt.compat import get_user_model
+
+from apps.api.models import APIKey
+
+
+class JWTKeyAuthentication(JSONWebTokenAuthentication):
+
+    def authenticate_credentials(self, payload):
+        """
+        Returns an active user that matches the payload's user id and email.
+        """
+        User = get_user_model()
+        try:
+            api_key = APIKey.objects.get(key=payload['iss'])
```

Comment:
[Line 16 of apps/api/jwt_auth/handlers.py](https://github.com/mozilla/olympia/pull/808/files#diff-e93a80d27d00ba5f9f8c4fa95560b692R16) seems very similar but that uses `APIKey.objects.get(key=key, type=SYMMETRIC_JWT_TYPE)`. Why the difference? Should there be a helper on `APIKeyManager` like `APIKey.objects.get_key(key)`?